<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JournaldConfigurationWidget</class>
 <widget class="QWidget" name="JournaldConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxOptions">
     <property name="title">
      <string>Journald options</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="lastBootOnly">
        <property name="text">
         <string>Display log entries from current boot only</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="entriesTypeComboBox">
        <item>
         <property name="text">
          <string>Display all log entries</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Display log entries of processes of the current user</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Display log entries of system services and the kernel</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxRemoteJournals">
     <property name="title">
      <string>Remote journals</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="remoteJournalsListWidget">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Address</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Port</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">HTTPS</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="buttonsGridLayout">
        <item row="3" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="addAddressButton">
          <property name="text">
           <string>Add address</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="removeAddressButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Remove address</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="modifyAddressButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Modify address</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
